README.md

```diff
@@ -270,24 +270,12 @@ Golang agent supports the following dynamic configurations.
 
 This feature is used in cooperation with the [skywalking-rover](https://github.com/apache/skywalking-rover) project.
 
-When go2sky keeps alive with the backend, it would write a metadata file to the local (temporary directory) at the same time, which describes the information of the current process.
-The rover side scans all processes, find out which process contains this metadata file. Finally, the rover could collect, profiling, with this process.
-
-### Metadata File
```

Comment:
I think mentioning metadata file still makes sense?

Reply:
Sure, I have added it back.